lib/markdoc/version.rb

```diff
@@ -1,3 +1,3 @@
 module Markdoc
-  VERSION = '1.2.0'
+  VERSION = '1.3.0'
```

Comment:
@GrahamJerome I guess here we are dealing with a patch version update, like `1.2.1`.

Reply:
@ochko oh yes, that makes sense. I've added that. Thanks!